db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
+      t.string :host_name
```

Comment:
Is this like the literal name of a Host, or a hostname like the part of a URL?  If the latter can you name is `hostname` for consistency?  If the former, then it's fine as is.